tests/test_ops/test_roi_align.py

```diff
@@ -30,66 +31,67 @@
                [0.42968750, 0.39062500, 0.39062500, 0.03906250],
                [0.04296875, 0.03906250, 0.03906250, 0.00390625]]]])]
 
+pool_h = 2
```

Comment:
For input and output tensors, yapf will make the format ugly. We can disable it temporally.